assets/variant-list.css

```diff
@@ -215,9 +215,11 @@ variant-list-remove-button {
 
   .variant-list-total__column {
     display: flex;
```

Comment:
Also for generic errors (at the bottom) well have to ensure they show even when there is no bottom bar